biddings/views.py

```diff
@@ -1 +1,72 @@
-from django.shortcuts import render
\ No newline at end of file
+import json
+
+from django.http  import JsonResponse
+from django.views import View
+
+from cores.utils     import login_decorator
+from users.models    import User
+from biddings.models import Order, Bidding
+from products.models import ProductSize, Size
+
+class OrderView(View):
+    @login_decorator
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            user = request.user
+
+            product   = data['product']
+            size_name = data['size']
+            price     = data['price']
+            is_buyer  = data['is_buyer']
+
+            size         = Size.objects.get(name = size_name).id
+            product_size = ProductSize.objects.get(product_id = product, size_id = size).id
```

Comment:
```suggestion
            if ProductSize.objects.filter(size__name = size, product_id = product).exists():
                return JsonResponse({"message" :  "product_size does not exist"}, status=204)
                
            product_size = ProductSize.objects.get(size__name = size, product_id = product).id
```
1. objects.get을 사용하기 전에 예외처리 추가해 주세요.
2. id를 찾는 과정 없이 size의 이름을 가지고 바로 데이터를 가져올 수 있습니다.

Reply:
반영 완료했습니다!